### DISCUSSION TITLE
Graylog + ejbca docker

### DISCUSSION THREAD

**Discussion by thr35h** [Keyfactor/ejbca-ce#144] [Q&A]
Hi

I'm having trouble trying to get Graylog to work with the ejbca docker container. I have it up and running persistently but when I edit the WildFly standalone.xml file for my needs and restart it, the standalone.xml seems to be overwritten everytime. Does anyone have any ideas or can tell me what I'm probably doing wrong?

**Comment by primetomas** [comment]
I believe this is how containers work in general, it's not specific to the EJBCA container typically. With containers you typically don't want to modify the application, like you would do if it's running in a VM. The log output from the container is sent to stdout I think, and you typically with with the container runtime to capture logs from various containers in your system and gather it. This actually helps you to not have to bother with specific logging configuration on each and every application you run as a container, but you leverage the container infrastructure to do this for you.
Docker have some documentation on logging here: https://docs.docker.com/config/containers/logging/

**Comment by pkmarios** [comment]
As Tomas mentioned we should not alter the container standalone.xml. Several other ways could be used to ship the log to a syslog server. Here is an example using docker gelf logging driver: https://github.com/Keyfactor/ejbca-containers/tree/master/apps-integration/graylog
